src/main/java/org/checkerframework/checker/crypto/CryptoAnnotatedTypeFactory.java

```diff
@@ -29,6 +33,20 @@
     /** The @{@link UnknownAlgorithmOrProvider} annotation. */
     protected final AnnotationMirror UNKNOWNALGORITHMORPROVIDER =
             AnnotationBuilder.fromClass(elements, UnknownAlgorithmOrProvider.class);
+    /** The Annotation Class.AllowedAlgorithms() method. */
```

Comment:
```suggestion
    /** The {@code AllowedAlgorithms.value()} method. */
```

Reply:
I'm not sure how to parse the `Annotation Class.AllowedAlgorithms()`. Does my suggested change make sense?

Reply:
Let me see...

Reply:
Or

```
/** The {@link AllowedAlgorithms#value} element/argument. */
```